docs/source/evaluations/fine_tuning.rst

```diff
@@ -38,14 +41,178 @@ Benchmark: Places205
 
 The configuration for full fine-tuning on Places205 is available in `benchmark/fulltune/places205 <https://github.com/facebookresearch/vissl/tree/master/configs/config/benchmark/fulltune/places205>`_ and can be run as follows:
 
-
 .. code-block:: bash
 
     python tools/run_distributed_engines.py \
       config=benchmark/fulltune/places205/eval_resnet_8gpu_transfer_places205_fulltune \
       config.MODEL.WEIGHTS_INIT.PARAMS_FILE=<my_weights.torch>
 
 
+Benchmark: CIFAR-100
```

Comment:
I think we can move the low-shot section out to a dedicated `low_shot_transfer.rst` to make it more visible :)

Reply:
> can you verify once we need to make changes to docs/readmes etc due to the renaming of `low_shot_classification` ?

Indeed, to my surprise, I did not see any reference to the `low_shot_classification` folder. I will create a dedicated documentation as you suggests, it would indeed make sense to separate it from the full fine-tuning, and we can then enrich it with the two others tasks (VOC07 and Places205).

Reply:
I moved the documentation in a specific file in 27f53fe.

I propose we complete it with the VOC07 and Places205 in another PR (I first need to play with these configuration to better understand them): I can create an issue with it and add it to my backlog.

Reply:
sounds great to me :)